packages/hosted-form-v2/src/hosted-field.ts

```diff
@@ -60,7 +60,7 @@ export default class HostedField {
     private getFrameSrc(orderId?: number): string {
```

Comment:
Looks ok for now, but maybe we need to think of passing a `client` property here by which we can serve the relevant source.

Reply:
@animesh1987 Yes, agree. When we want to implement hosted fields v2 on the checkout, this change would be mandatory.